admin-ui/src/pages/settings-page.tsx

```
@@ -1389,6 +1395,27 @@ function AdvancedSettingsCard({
+          </Label>
+          <Input
+            type="number"
+            min="0"
+            step="0.5"
+            value={modelRefreshIntervalInput}
+            onChange={(e) => onModelRefreshIntervalChange(e.target.value)}
+          />
+          <div className="text-muted-foreground text-xs">
+            {t("settingsPage.advanced.modelRefreshIntervalHint")}
+          </div>
+          {modelRefreshIntervalIssue ? (
```

ESLint

[error] Delete `·(`

(prettier/prettier)